src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,170 @@
+import type { ReactElement } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { Box, Button, Grid, Stack } from '@mui/material';
+import { Search as SearchIcon } from '@mui/icons-material';
+
+import {
+  AgeTextField,
+  BiomarkersAutocomplete,
+  CancerStageAutocomplete,
+  CancerSubtypeTextField,
+  CancerTypeTextField,
+  ECOGScoreAutocomplete,
+  KarnofskyScoreAutocomplete,
+  MedicationsAutocomplete,
+  MetastatsisTextField,
+  RadiationAutocomplete,
+  SurgeryAutocomplete,
+  TravelDistanceTextField,
+  ZipcodeTextField,
+} from './FormFields';
+import SearchImage from '@/assets/images/search.png';
+import type { Patient } from '@/utils/patient';
+import MatchingServices from './MatchingServices';
+import { SearchFormValuesType } from './types';
+
+export type SearchFormProps = {
+  patient: Patient;
+};
+
+const SearchForm = ({ patient }: SearchFormProps): ReactElement => {
+  const router = useRouter();
+
+  const defaultValues: Partial<SearchFormValuesType> = {
+    age: patient.age,
+    cancerType: patient.cancerType,
+    travelDistance: '100',
+    zipcode: patient.zipcode,
+  };
+
+  const { handleSubmit, control } = useForm<SearchFormValuesType>({ defaultValues });
+  const onSubmit = data => router.push({ pathname: '/results', query: data });
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Box bgcolor="grey.200">
+        <Box p={{ xs: 0, md: 2 }} display={{ xs: 'none', md: 'block' }}>
+          <Stack alignItems="center" direction={{ xs: 'column', lg: 'row' }} justifyContent="center">
+            <Box>
+              <Image
+                src={SearchImage}
+                alt="Clinical Trial Finder Search"
+                layout="fixed"
+                width={400}
+                height={190}
+                priority
+              />
+            </Box>
+
+            <Box ml={10} textAlign={{ xs: 'center', lg: 'left' }}>
+              <Box fontSize={{ xs: 30, lg: 38, xl: 63 }} fontWeight={300}>
+                Let's find some clinical trials
+              </Box>
+
+              <Box color="grey.600" fontSize={{ xs: 20, lg: 25, xl: 28 }} fontWeight={300}>
+                Search with data populated from your record, or change to find matching trials
+              </Box>
+            </Box>
+          </Stack>
+        </Box>
+
+        <Grid columns={8} container spacing={2} px={2} py={{ xs: 0, md: 2 }} mt={{ xs: 0, md: -2 }}>
```

Comment:
```suggestion
        <Grid columns={8} container spacing={2} px={2} py={{ md: 2 }} pb={{ xs: 2 }} mt={{ xs: 0, md: -2 }}>
```
Maybe, just so the search button has some space before the end of the page? On medium/large screens it appears that there's a bottom padding of 2.

Current: 
![image](https://user-images.githubusercontent.com/33106214/133855894-bc312797-7c80-4152-817c-fc19e31fe2dc.png)

With suggestion:
![image](https://user-images.githubusercontent.com/33106214/133855922-f33daa3a-d96f-41b5-a76d-0c90bbb299df.png)

Reply:
I like it!